test/Microsoft.AspNet.Mvc.TagHelpers.Test/OptionTagHelperTest.cs

```diff
@@ -233,7 +236,7 @@ public class OptionTagHelperTest
                             "text")
                     },
                     {
-                        "HtmlEncode[[text]]", null, null, new [] { "text", },
+                        "HtmlEncode[[text]]", null, null, new CurrentValues(new HashSet<string>(new [] { "text", })),
```

Comment:
Forgot to add a similar entry using `"text"` as the element's content e.g. the following will now also generate `selected="selected"`:
``` c#
  "text", null, null, new CurrentValues(new HashSet<string>(new [] { "text", })),
  ...
```